### ISSUE TITLE
incremental rendering in development mode

### ISSUE THREAD

**Issue by milahu** [evnbr/bindery#129]
say i have a book with 1000 pages, and i work on page 4

bindery takes some time to render all pages
for my development loop, i want to render fast

so i want bindery to render only pages 1 2 3 4
because only these pages affect the layout on page 4

incremental rendering should be optional
as probably, this would break printing

idea: parse `document.location.hash` for something like `bindery=incremental`

**Comment by evnbr** [comment]
Yeah, I would love to get this working for that fast feedback loop. My end goal would be possible to do things like let you adjust some CSS in web inspector, and then incrementally update the preview, so it feels really like web design.
